src/xenia/kernel/xthread.h

```diff
@@ -30,6 +30,7 @@ class XEvent;
 constexpr uint32_t X_CREATE_SUSPENDED = 0x00000001;
 
 constexpr uint32_t X_TLS_OUT_OF_INDEXES = UINT32_MAX;
+constexpr uint32_t X_THREAD_MASK = 0xF8000000;
```

Comment:
Please add a note about how Burnout Paradise uses this. Also is THREAD_MASK the original constant name from the NT kernel? If not, it should be in the kName format, not X_NAME, and possibly not in this file.